main.asm

```diff
@@ -21,7 +21,7 @@
 
 %define ASMTTPD_VERSION "0.3"
 
```

Comment:
Yes apologies for that I was using that port for testing.
On 10 Jul 2016 1:47 a.m., "Nathan Torchia" <notifications@github.com> wrote:

> In main.asm
> <https://github.com/nemasu/asmttpd/pull/24#discussion_r70174360>:
>
> > @@ -21,7 +21,7 @@
> >
> >  %define ASMTTPD_VERSION "0.3"
> >
>
> Please keep it on port 80.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nemasu/asmttpd/pull/24/files/9843b1a77a00014e81a348dde1945f71b8e7bade#r70174360>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABSlctyCssJjLuOvUSO0FmOUbfAHelmeks5qUEEfgaJpZM4JIomQ>
> .
>